package/Dockerfile.dapper-base

```
@@ -39,7 +39,7 @@ RUN dnf -y distrosync --nodocs --setopt=install_weak_deps=False && \
                    gcc git-core curl moby-engine make golang kubernetes-client \
                    findutils upx jq golang-x-tools-goimports ShellCheck shflags && \
     dnf -y remove libsemanage && \
-    rpm -qa "selinux*" | xargs rpm -e --nodeps && \
+    rpm -qa "selinux*" | xargs rpm -r -e --nodeps && \
```

```suggestion
    rpm -qa "selinux*" | xargs -r rpm -e --nodeps && \
```

Sorry, this is the correct invocation.

---

OK we'll just squash it before merge :)